Userland/Applications/Maps/MapWidget.h

```diff
@@ -50,6 +50,30 @@ class MapWidget : public GUI::Frame {
         update();
     }
 
+    struct Panel {
+        enum class Position {
+            TopLeft,
+            TopRight,
+            BottomLeft,
+            BottomRight,
+        };
+        String text;
+        Position position;
+        Optional<URL> url {};
+        bool is_attribution { false };
```

Comment:
This is a bit of a leaky abstraction - I suggested adding a `bool` preventing it from being removed, but your version exposes an implementation detail about attribution panels. Maybe make it `bool persistent { false }`?